nautobot/docs/release-notes/version-2.0.md

```diff
@@ -14,4 +14,149 @@ If you are a user migrating from Nautobot v1.X, please refer to the ["Upgrading
 
 ### Removed
 
+#### Removed DCIM Shadowed Filter Fields ([#2804](https://github.com/nautobot/nautobot/pull/2804))
+
+As a part of breaking changes made in v2.0, shadowed filter/filterset fields are being removed in the dcim app:
+
+Currently for foreign-key related fields, **on existing core models in the v1.3 release train**:
+    - The field is shadowed for the purpose of replacing the PK filter with a lookup-based on a more human-readable value (typically `slug`, if available).
+    - A PK-based filter is available as well, generally with a name suffixed by `_id`
+
+For example:
+
+```python
+    site = django_filters.ModelMultipleChoiceFilter(
+        field_name="site__slug",
+        queryset=Site.objects.all(),
+        to_field_name="slug",
+        label="Site (slug)",
+    )
+    site_id = django_filters.ModelMultipleChoiceFilter(
+        queryset=Site.objects.all(),
+        label="Site (ID)",
+    )
+```
+
+Now these two fields will be replaced by a single field using `NaturalKeyOrPKMultipleChoiceFilter` introduced in v1.4:
+
+```python
+    from nautobot.utilities.filters import NaturalKeyOrPKMultipleChoiceFilter
+    site = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=Site.objects.all(),
+        label="Site (slug or ID)",
+    )
+```
+
+Below is a table documenting all such changes and where they occurred.
+
+| Filterset                      | Old Filter Field   | Changes                                                                      |
+|--------------------------------|--------------------|------------------------------------------------------------------------------|
+| LocatableModelFilterSetMixin   | region_id          | Deleted                                                                      |
```

Comment:
Referencing internal classes isn't very helpful to the end user. Would be better to enumerate the UI and REST API endpoints that are impacted by these changes.